addons/medical_gui/functions/fnc_updateActions.sqf

```diff
@@ -50,6 +50,16 @@ private _shownIndex = 0;
         _ctrl ctrlSetPositionY POS_H(1.1 * _shownIndex);
         _ctrl ctrlCommit 0;
 
+        // Show warning if tourniquet will interfere with action
+        if (((_category in ["examine", "medication"]) || (_items findIf {"IV" in _x}) > -1)
```

Comment:
Isn't there an IV category? The `_items findIf {"IV" in _x}) > -1` check is too specific imo.

Reply:
IVs are in the advanced category, shared with things like stitching and CPR. Checking the item name was the most succinct way I could figure out without having to pass extra information about the actions.